packages/wallets/src/components/Base/WalletDialog/WalletDialogHeader.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { PropsWithChildren } from 'react';
+import classNames from 'classnames';
+import { Dialog } from '@deriv-com/ui';
+
+type TProps =
```

Comment:
Using discriminated union to notify developer not forget to provide `onClose` props because by default we are rendering `Header` with close button.
And if the developer will pass `hideCloseIcon = true`, TS will not complain that onClose prop is needed.